circle.yml

```diff
@@ -76,6 +79,10 @@ test:
     # are much faster, no need to wait for them)
     - while [ ! -e ~/sauce_is_ready ]; do sleep 1; done
 
+    # Start taking screenshots in the background while the unit tests are running
+    - npm install wd && node script/screenshots.js http://localhost:3000/test/visual.html && touch ~/screenshots:
```

Comment:
playing with it at http://yamltojson.com/ , it looks like in YAML syntax, a line that starts with an open-brace is an object not a string and using an object as a key is a syntax error, but you can quote it:

```yaml
# Start taking screenshots in the background while the unit tests are running
- "{ test -d node_modules/wd || npm install wd; } && node script/screenshots.js http://localhost:3000/test/visual.html && touch ~/screenshots":
    background:true
```